applications/madini_tracechain.md

```diff
@@ -134,7 +151,7 @@ www.madinitracechain.com
 
 
 Madini MDN Token
-
```

Comment:
How it works

With Madini TraceChain you can easily upload your product descriptions, attributes, photos, certificates, documents and videos to the VeChainThor blockchain. The users of TraceChain will be rewarded on MDN token for using the Traceability system everytime a new roduct hash is generted, empowering small scale producers to directly benefit and get more services from the Madini ecosystem.

When the upload is completed, the system will generate MDN to the users wallet and an unique QR code assigned to the product. You will also be able to see the transaction (TX) ids that your business has generated, giving you a full overview of the number of TXs that your products have produced accross the product’s journey.

OPEN
There’s no need of a special APP or device to read your QR codes and verify your products. Any QR reader installed on your phone will detect and show the traceability information

PROFITABLE
Traceability has become an essential task for producers and exporters, Supplying adequate quality and safety assurance open new markets and adds vaue to your products. Usage of the platform automatically generates MDN also reward the platform users, to serve as a multi-purpose token rewarding early supporters of the project and create true value for our community.

AVAILABLE
the entire story of your product including proof of authenticity and origin, all in a consumer friendly interface worldwide

STABLE AFFORDABLE
VeChainThor provides security, scalability and, more importantly, affordability. Madini dApps running on VeChainThor are guaranteed stable and predictable fees. 

MDN is the Madini VeChain token that is used to carry value or “smart money” from smart contracts. In other words, transactions on decentralized applications occurring on Madini VeChain's blockchain will use MDN. It is available for investing by the general public. MDN token is the Madini utility token in our Ecosystem since it is linked to the Madini TraceChain platform, the future Madini Marketplace, to the Madini Wallet - Android APP - and other Madini dApps. MDN Token is directly rewarded to the Madini TraceChain users (100MDN/product) and can be used for trading/ transact on Madini Ecosystem as well as the many other use cases that accept this electronic asset as mode of payment. 

MDN Distribution: 20% angel investors and initial shareholders, 5% developers, 75% available public pool